packages/utils/src/ts-eslint/Rule.ts

```diff
@@ -220,20 +220,42 @@ interface RuleContext<
    * Returns the current working directory passed to Linter.
    * It is a path to a directory that should be considered as the current working directory.
    * @since 6.6.0
+   * @deprecated Use {@link `cwd`} instead.
```

Comment:
we probably want to remove these `@deprecated` tags.
most plugins will support ESLint v7 and v8 - they're not limited to 8.40.0
this means that trying to direct consumers to use these new APIs is an impossible state as they have to do things like `context.cwd ?? context.getCwd()` - which is just worse code than `context.getCwd()`.

Reply:
You are 100% right - shall I open a new PR?

Reply:
Hmm, that makes sense.

@so1ve if you want to get another PR to typescript-eslint under your belt 😄 we can merge it any time before Monday midday EST. I'll set a reminder to send one in if one hasn't yet been by then.